_includes/search-lunr.html

```diff
@@ -0,0 +1,71 @@
+<script src="/js/lunr.js"></script>
+
+<script>
```

Comment:
I'll be the bad cop and ask if it makes sense to create a separate js file, say àssets/lunr-search.js` that would handle the index? However if this is only used once, and the whole file is small and readable, it's fine as it is, too.

Reply:
Sure I'll move it to `/assets/js/vendor`